Jenkinsfile

```diff
@@ -57,23 +57,32 @@ node('ios-osx') {
 
         stage('Run Tests') {
             withEnv(['platformName=ios', 'udid=8F1A1F1B-4428-45F4-B282-DE628D9A54A1', 'deviceName=iPhone 6', 'platformVersion=10.3.1']) {
-                timeout(20) {
+                timeout(45) {
                     sh '''#!/bin/bash -l -x
                         cd external/autobots
                         chmod 0755 requirements.txt
                         sudo -H pip install -r requirements.txt
-                        python testRunner.py | true
+                        python testRunner.py
+                        ret=$?
+                        if [ $ret -ne 0 ]; then
```

Comment:
I would rather go with `set -e` for all `sh` steps which https://github.com/cliqz-oss/browser-ios/pull/305 is introducing